torch_geometric/nn/models/schnet.py

```diff
@@ -112,9 +112,9 @@ def __init__(
         self.num_interactions = num_interactions
         self.num_gaussians = num_gaussians
         self.cutoff = cutoff
-        self.readout = readout
+        self.aggr = aggr_resolver(readout)
+        self.sum_aggr = SumAggregation()
         self.dipole = dipole
-        self.readout = 'add' if self.dipole else self.readout
```

Comment:
Could you clarify this change? Shouldn't we resolve the aggr of `readout` after this?

Reply:
yes, good spot! I'll fix this soon